addons/mrp/models/stock_move.py

```diff
@@ -401,3 +401,10 @@ def _update_quantity_done(self, mo):
             self.move_line_ids = self._set_quantity_done_prepare_vals(new_qty)
         else:
             self.quantity_done = new_qty
+
+    def _update_candidate_moves_list(self, candidate_moves_list):
+        super()._update_candidate_moves_list(candidate_moves_list)
+        for production in self.mapped('raw_material_production_id'):
```

Comment:
As candidate_move_list is a list, it do not remove duplicates. If you have all the `n` moves of a production order in `self`, you will add `n²` items in the list. We should either use a set or at least make 
```python
candidate_moves_list.append(self.raw_material_production_id.move_raw_ids)
```

Reply:
This is complete bullshit. You can forget it :shameonme